test/lib/RuleManager.test.ts

```diff
@@ -222,15 +222,15 @@ describe('RuleManager', () => {
 						results = JSON.parse(output);
 					}
 
-					expect(results).to.be.an("array").that.has.length(4);
+					expect(results, JSON.stringify(results)).to.be.an("array").that.has.length(3);
 					for (const result of results) {
 						expect(result.violations[0], `Message is ${result.violations[0]['message']}`).to.have.property("ruleName").that.is.not.null;
 					}
 				});
 
 				it('Filtering by multiple categories runs any rule in either category', async () => {
 					// Set up our filter array.
-					const filters = [new RuleFilter(FilterType.CATEGORY, ['Best Practices', 'Design'])];
+					const filters = [new RuleFilter(FilterType.CATEGORY, ['Best Practices', 'Error Prone'])];
```

Comment:
This is the reason for adding the `Error Prone` category to the catalog. The previous revision had `4` as the expected results for `['Best Practices']` above as well as ['Best Practices', 'Design']` here.